README.md

```diff
@@ -81,7 +81,7 @@ The map of containers consists of the name of the container mapped to the contai
 	* `device` (array) Add host devices.
 	* `dns` (array)
 	* `entrypoint` (string)
-	* `env` (array)
+	* `env` (array/mapping) `env` can be declared as a mapping where each `key`-`value` will be translated to a string `"key=value"`.
```

Comment:
@adrianhurt I like your latest suggestion. If you update the text I'll merge this in.